### DISCUSSION TITLE
How to configure Docker as Container Runtime?

### DISCUSSION THREAD

**Discussion by davidmcnamee** [kube-hetzner/terraform-hcloud-kube-hetzner#595] [Q&A]
I'm looking to set up [docker outside of docker (DooD)](http://tdongsi.github.io/blog/2017/04/23/docker-out-of-docker/) so that I can use the fastest-possible docker to build images inside a k8s job. If I understand correctly, this requires that I use docker as my container runtime so that my pods can communicate (via domain socket) to the docker engine running "outside" of the cluster. [This k3s doc says it's possible](http://tdongsi.github.io/blog/2017/04/23/docker-out-of-docker/), but I wasn't sure how to implement that with kube-hetzner. Can anyone help me out here?

**Comment by mysticaltech** [comment]
@davidmcnamee Maybe try SSH'ing into a node (see readme), you have crictl there and other cool tools, it's also possible that the docker command works, but have not tried. 

If you need to install new packages on the OS, you can during cluster deploy with the extra_packages variables.
